python/ray/rllib/agents/agent.py

```diff
@@ -262,7 +262,7 @@ def _setup(self, config):
                                     self._allow_unknown_subkeys)
         self.config = merged_config
         if self.config["log_level"]:
-            RLLIB_LOGGER.setLevel(self.config["log_level"])
+            logging.getLogger("ray.rllib").setLevel(self.config["log_level"])
```

Comment:
I guess one odd issue would be when you do two agents and set one to have more verbose logging but it ends up doubling your output...

Reply:
Yeah but there's not much we can do about that. Two agents in the same process is kind of an antipattern anyways.